src/test/span_tests.cpp

```diff
@@ -42,9 +42,9 @@ using namespace spannable;
 
 BOOST_AUTO_TEST_SUITE(span_tests)
 
-// Make sure template Span template deduction guides accurately enable calls to
-// Span constructor overloads that work, and disable calls to constructor overloads that
-// don't work. This makes it is possible to use the Span constructor in a SFINAE
+// Make sure template std::span template deduction guides accurately enable calls to
+// std::span constructor overloads that work, and disable calls to constructor overloads that
+// don't work. This makes it is possible to use the std::span constructor in a SFINAE
 // contexts like in the Spannable function above to detect whether types are or
 // aren't compatible with Spans at compile time.
```

Comment:
Seems like a leftover `Spans`

Reply:
Done, you can resolve it (except that "[As a general rule, we never use an apostrophe in writing plural forms](https://www.sussex.ac.uk/informatics/punctuation/apostrophe/plurals)")